### DISCUSSION TITLE
Changing the position of Ask AI

### DISCUSSION THREAD

**Discussion by Gitstar-OC** [fuma-nama/fumadocs#961] [Show and tell]
Hey, Instead of keeping the position of Ask AI in the docs like this 

![Screenshot 2024-10-22 172024](https://github.com/user-attachments/assets/ca636e4a-7767-4aee-ac6c-938a12c2bc57)

why don't you use the below one for ai, it will help in the ui, 
![Screenshot 2024-10-22 172123](https://github.com/user-attachments/assets/d2ec484d-4837-4f62-988d-f84527ba3dfb)

`Position changed from left top corner to right bottom corner`

**Comment by fuma-nama** [comment]
I hate a button that blocks the content, it should be on the navbar.

**Comment by Gitstar-OC** [comment]
Then, maybe it will look more better good near the sidebar button, the name of the website will look more better at the top. How does this look ? 

### The website name will be as it is
![Screenshot 2024-10-22 184540](https://github.com/user-attachments/assets/9a7ee55b-0752-4feb-a7d4-82eddbd730b2)

###  The footer of the sidebar may have various options on looks on what you may choose`
![Screenshot 2024-10-22 184547](https://github.com/user-attachments/assets/b89222b7-c3a1-46bb-8f25-ccff1308855d)

![Screenshot 2024-10-22 184653](https://github.com/user-attachments/assets/5e580535-78ee-4990-ada0-9675e63f78de)

###  We will use this theme changer, the one shadcn ui has for balanced design`
![Screenshot 2024-10-22 183101](https://github.com/user-attachments/assets/d32f22fd-d36e-4da9-9b4c-686fe3be4384)

### And for the AI button component we can use this styling
![image](https://github.com/user-attachments/assets/dcb71104-c438-416f-b688-f56e8c1e8bb0)

 What do you think about it ?

**Comment by Gitstar-OC** [comment]
This will look like this, much better 
![image](https://github.com/user-attachments/assets/a63a403c-401c-4eb9-9783-bc5178b8afb1)

**Reply by fuma-nama** [reply]
I prefer to put navigation functionality to sidebar head, and the rainbow color doesn't really fit our design. Maybe you can use the search AI button on your docs and customise it as you want, but I won't change it on our official site

**Reply by Gitstar-OC** [reply]
![image](https://github.com/user-attachments/assets/0535bb4b-0fd5-41a6-b102-fa767343b559) 

So you will let this as it is or change it a little bit in the navbar ?

**Reply by fuma-nama** [reply]
I would love to keep the current design. 

Your suggestion is good but it doesn't have a convincing reason and advantage/benefit over the original one. Thanks for your suggestion too!
